saleor/graphql/app/mutations.py

```diff
@@ -97,6 +97,26 @@ def clean_instance(cls, info, instance):
             raise ValidationError({"id": ValidationError(msg, code=code)})
 
 
+class AppTokenVerify(BaseMutation):
+    valid = graphene.Boolean(default_value=False, required=True)
```

Comment:
No description 👮‍♂️

Reply:
There is still a lack of description. 👮‍♀️

Reply:
👮‍♀️ 👮‍♂️

Reply:
I forgot to push the changes :O